viscy/representation/evaluation/clustering.py

```diff
@@ -150,3 +150,32 @@ def clustering_evaluation(embeddings, annotations, method="nmi"):
         raise ValueError("Invalid method. Choose 'nmi' or 'ari'.")
 
     return score
+
+
+def compute_track_msd_statistics(
+    msd_per_tau: dict[int, list[float]],
+) -> tuple[dict[int, float], dict[int, float]]:
+    """
+    Compute MSD statistics (mean and std) for a single track.
+
+    Parameters
+    ----------
+    features : ArrayLike
+        Feature matrix (n_timepoints, n_features) for a single track
+    timepoints : ArrayLike
+        Time points corresponding to each feature vector
+    metric : str, optional
+        Distance metric to use, by default "euclidean"
```

Comment:
Are these correct?

Reply:
deprecated we dont use these functions anymore